src/components/ScholarshipForm.jsx

```diff
@@ -168,7 +203,90 @@ function ScholarshipForm({ setSubmissionAlert }) {
     </Grid>
   );
 
-  stepperItems.Requirements = 'requirements';
+  stepperItems['Eligibility Requirements'] = (
+    <Grid container spacing={3}>
+      <Grid item xs={12}>
+        <Typography className={classes.stepperDescription}>
+          Include information that is required for applicants to have.
+        </Typography>
+      </Grid>
+
+      <Grid item xs={12}>
+        <FormControlLabel
+          control={
+            <Checkbox
+              checked={hasReqs}
+              onChange={(event) => setHasReqs(event.target.checked)}
+              color="primary"
+            />
+          }
+          label="NO ELIGIBILITY REQUIREMENTS"
+        />
+      </Grid>
+
+      <Grid item sm={6} xs={12}>
+        <FormikMultiSelect
+          label="Grade"
+          labelStyle={classes.inputLabel}
+          value={grades}
+          options={gradeOptions}
+          changeFn={setGrades}
+        />
+      </Grid>
+
+      <Grid item sm={6} xs={12}>
+        <FormikTextField
+          label="GPA"
+          id="gpa"
+          formik={formik}
+          labelStyle={classes.inputLabel}
+        />
+      </Grid>
+
+      <Grid item sm={6} xs={12}>
+        <InputLabel className={classes.inputLabel}>School</InputLabel>
+        <Autocomplete
```

Comment:
I suggest making these autocompletes [freeSolo](https://mui.com/components/autocomplete/#free-solo) and allow [multiple](https://mui.com/components/autocomplete/#free-solo) values

Reply:
yes, I was just testing some things.